modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/manager/DefaultPortalKaleoManager.java

```diff
@@ -321,6 +322,8 @@ private String _getLocalizedTitle(long companyId, String definitionName)
 		return _localization.getXml(localizedValuesMap, "title");
 	}
 
+	private static final String _DEFINITION_ERC = "L_SINGLE_APPROVER";
```

Comment:
@mxs2 rename to `_WORKFLOW_DEFINITION_ERC` and rename _DEFINITION_NAME to `_WORKFLOW_DEFINITION_NAME`. Thx!